libs/atqr/domain/src/lib/EmailService/interfaces/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './email.interface';
+export * from './mailer.interface';
```

Comment:
You're exporting this "mailer interface", but I don't see you using it. Is it supposed to be used eventually?